config/config.exs

```diff
@@ -15,3 +15,17 @@ config :logger, :console,
   metadata: [:request_id]
 
 import_config "#{Mix.env}.exs"
+
+# %% Coherence Configuration %%   Don't remove this line
+config :coherence,
+  user_schema: CoursePlanner.User,
+  repo: CoursePlanner.Repo,
+  module: CoursePlanner,
+  logged_out_url: "/",
+  email_from_name: "Your Name",
+  email_from_email: "yourname@example.com",
```

Comment:
I think it would be good to get `email_from_name` and `email_from_email` from the environment